Formula/zig.rb

```diff
@@ -4,6 +4,7 @@ class Zig < Formula
   url "https://ziglang.org/download/0.9.1/zig-0.9.1.tar.xz"
   sha256 "38cf4e84481f5facc766ba72783e7462e08d6d29a5d47e3b75c8ee3142485210"
   license "MIT"
+  revision 1
   head "https://github.com/ziglang/zig.git"
```

Comment:
it would be actually great to specify the default branch, but I think we can followup in the next run.